### PR TITLE
feat: Tauri RSS feeds

### LINKED ISSUE #1845: Add RSS and Atom support for beta.tauri.app's blogs [feat]
### Describe the problem

I can't retrieve any rss.xml files from the new beta.tauri.app's website.

### Describe the solution you'd like

Either add new blogs since 15 June 2023 to tauri.app or add RSS and Atom support for beta.tauri.app.

### Alternatives considered

_No response_

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by vasfvitor**
- Closes #1845 #916 
- What does this PR change? Add Blog RSS Feed

---
TODO: 
~~- Commit pnpm lockfile~~
- Fix feed hour 00:00:00 -> maybe? https://github.com/simonhyll/tauri-by-simon/blob/main/src/pages/feed.xml.js

Is having the time as 00:00:00 a issue? The above solution may work but I'd like to keep it simple. Or maybe add it when more feeds are created, like docs.xml, for example.

Preview:
image
image
image

### CODE REVIEW

File: src/pages/blog.xml.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+			if (dateA < dateB) return 1;
+			if (dateA > dateB) return -1;
+			return 0;
+		} else return 0;
+	});
+
+	return rss({
+		title: 'Tauri Blog',
+		description: 'The cross-platform app building toolkit',
+		// @ts-expect-error
```
Comment (dreyfus92) [inline src/pages/blog.xml.ts]: Why is TypeScript yelling here?

**Review by dreyfus92 — COMMENTED** [review]
Awesome job @vasfvitor, just left you a question 😁

File: src/pages/blog.xml.ts (hunk 1 above)
Comment (vasfvitor) [reply src/pages/blog.xml.ts]: oh, good question. It says that "Type 'URL | undefined' is not assignable to type 'string | URL'."
I could throw a `context.site as URL`. What do you think?

Comment (dreyfus92) [reply src/pages/blog.xml.ts]: That should do the work or even a typeguard. W/e you decide works for me 😀

**Comment by vasfvitor** [comment]
@simonhyll should I then add the other feeds in this PR?

**Comment by vasfvitor** [comment]
oh, I forgot to actually create the RSS button that links to it, I'll get to it

**Comment by vasfvitor** [comment]
done

**Review by simonhyll — APPROVED** [review]

**Review by dreyfus92 — APPROVED** [review]
🚢🚢🚢🚢🚢🚢

**Pull request merged**